src/js/lib/monster.ui.js

```diff
@@ -1350,7 +1350,16 @@ define(function(require) {
 			});
 
 			$.validator.addMethod('phoneNumber', function(value, element) {
-				return this.optional(element) || monster.util.getFormatPhoneNumber(value).isValid;
+				var isString = _.isString(value),
+					isArray = _.isArray(value),
+					isStringValid = isString && monster.util.getFormatPhoneNumber(value).isValid,
+					isArrayValid = isArray
+						&& _
+							.chain(value)
+							.map(monster.util.getFormatPhoneNumber)
+							.every('isValid')
+							.value();
+				return this.optional(element) || isStringValid || isArrayValid;
```

Comment:
Nice, that's functional. Maybe we could improve this block by writing our check only once (right now it is written twice, for each type).

To do so, we could wrap `value` into an array no matter what and flatten it, such as:

```js
return this.optional(element) || _
	.chain([value])
	.flatten()
	.map(monster.util.getFormatPhoneNumber)
	.every('isValid')
	.value();
```